### PR TITLE
docs: add comprehend doc

### PR DESCRIPTION
**Pull request by DenovVasil**
## Description
add comprehend documentation for [issue](https://github.com/camunda/connectors/issues/2924) 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by christinaausley** [comment]
Hi @DenovVasil -- do you mind providing a link to the related issue? Just let us know when you are ready for final review from @camunda/tech-writers, and an engineering review.

**Comment by DenovVasil** [comment]
> Hi @DenovVasil -- do you mind providing a link to the related issue? Just let us know when you are ready for final review from @camunda/tech-writers, and an engineering review.

Hi,  I added issue link to description
I will ask for your review after I receive confirmation with technical aspects.

**Comment by akeller** [comment]
@DenovVasil are you close to moving this out of draft so we can review and merge? We are prepping to cut the docs and you may need to backport these changes.

**Review by christinaausley — DISMISSED** [review]

**Review by sbuettner — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Review by christinaausley — DISMISSED** [review]
I have backported this for you @DenovVasil -- may we merge?

**Comment by christinaausley** [comment]
Note that based on Slack conversation, @DenovVasil is making a few adjustments and planning to merge shortly.

File: docs/components/connectors/out-of-the-box-connectors/amazon-comprehend.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+---
+id: amazon-comprehend
+title: Amazon Comprehend Connector
+sidebar_label: AWS Comprehend Connector
+description: Comprehend uses natural language processing (NLP) to extract insights about the content of documents.
```
[the remark this answers is not in the document]
Comment (conceptualshark) [reply docs/components/connectors/out-of-the-box-connectors/amazon-comprehend.md]: A little late here, sorry, but I agree with you! I try to edit anything like this so it's more our voice/understanding/use case and less word-for-word. I've run into this with the Kubernetes docs as well.

**Comment by DenovVasil** [comment]
> Note that based on Slack conversation, @DenovVasil is making a few adjustments and planning to merge shortly.

I consulted with my team and we decided that version 8.6.0 is not correct it should be 8.7.0

**Comment by christinaausley** [comment]
> > Note that based on Slack conversation, @DenovVasil is making a few adjustments and planning to merge shortly.
> 
> I consulted with my team and we decided that version 8.6.0 is not correct it should be 8.7.0

Should this go out with the `8.7` release then? If so, please update the labels. @DenovVasil

**Comment by christinaausley** [comment]
After jumping on a call with @DenovVasil, this can be merged into `next`, but should be removed from `8.6`. Updating this now, and preparing to merge.

**Pull request merged**